src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1641,6 +1641,495 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder", "t
         super().unfuse_lora(components=components)
 
 
+class AuraFlowLoraLoaderMixin(LoraBaseMixin):
+    r"""
+    Load LoRA layers into [`AuraFlowTransformer2DModel`] Specific to [`AuraFlowPipeline`].
+    """
+
+    _lora_loadable_modules = ["transformer", "text_encoder"]
+    transformer_name = TRANSFORMER_NAME
+
+    @classmethod
+    @validate_hf_hub_args
+    # Copied from diffusers.loaders.lora_pipeline.StableDiffusionLoraLoaderMixin.lora_state_dict
+    def lora_state_dict(
```

Comment:
I think we don't think we need to copy-paste it from `StableDiffusionLoraLoaderMixin`. 

It should be copy-pasted from 
https://github.com/huggingface/diffusers/blob/a5f35ee4731b731d6bd8977525873b0bc480cb42/src/diffusers/loaders/lora_pipeline.py#L1149

Reply:
This leads to the following test failure:

```
FAILED tests/lora/test_lora_layers_af.py::AuraFlowLoRATests::test_unexpected_keys_warning - ValueError: too many values to unpack (expected 2)
```